spec.md

```diff
@@ -25,6 +25,10 @@ More formally, the structure of such a file is:
 * Optional: tile content of first overview level
 * Tile content of full resolution image.
 
+The IFDs shall be linked by the pointers (the offset of the first IFD in the file header and the offset of the next IFD in each IFD) so that the full resolution image is first, followed by the overviews.
+
+Each IFD representing an overview shall have the tag NewSubfileType (254) with a value of 1, indicating the "reduced resolution image" subfile type.
```

Comment:
IFDs representing a mask should have value 4
IFDs representing the overview of a mask should have value 5

Reply:
Thanks, I'll study this suggestion and update the pull request.